test/e2e/hooks_test.go

```diff
@@ -19,7 +20,11 @@ type HooksSuite struct {
 	fixtures.E2ESuite
 }
 
-func (s *HooksSuite) TestWorkflowLevelHooks() {
+/*
+# Temporarily disable some tests until the following issues are resolved.
+# https://github.com/argoproj/argo-workflows/issues/10743
```

Comment:
Since this PR would break this test, instead of commenting these out, it would be better if you could fix this issue first. WDYT?

Reply:
sure! i opened new PR https://github.com/argoproj/argo-workflows/pull/10758.